demo/android/Activity/porcupine-activity-demo-app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name">Porcupine</string>
+    <string name="app_name">Porcupine (Android)</string>
```

Comment:
If we're trying to identify the app, maybe call it "Porcupine Android Activity Demo"?